docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -0,0 +1,32 @@
+options:
+  - title: Microservices
+    url: docs/rpc
+
+    nested_options:
+
+      - title: Quickstart
```

Comment:
Perhaps we can remove this section since it's no longer providing useful information, pretty much is showing the same menu that the user can find on the left menu.

Reply:
When going from the `Home` to the `Documentation` section, we can link directly to the `Core concepts` section.